### PR TITLE
docs: update global readme.md with `azd auth login` additional params

### PR DESCRIPTION
**Pull request by ShatilKhan**
I have added a `flag` to the wiki.
According to this issue #307 , `azd auth login` may not work sometimes while logging into `Codespaces`. So we may have to use the use `--use-device-code` flag.

It did work for me without the flag. But if it doesn't for someone, they can use the flag.

Previous wiki: 
```
# Login to azd. Only required once per install.
azd auth login
```

Updated
```
# Login to azd. Only required once per install.
azd auth login

# Use Flag if login does not work
azd auth login --use-device-code
```

Added the extra line, since it Does work sometimes. For other times, use the flag ⛳

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -247,6 +247,9 @@ This project uses [Azure Developer CLI (`azd`)](https://aka.ms/azd) to provision
 # Login to azd. Only required once per install.
 azd auth login
 
+# Use Flag if login does not work
```
Comment (anfibiacreativa) [inline README.md:250]: Hi @ShatilKhan . I think maybe we can be a bit more specific about what doesn't work. I think the issue is not the login/authentication not working, but the fact that it doesn't open a browser window? A link to the official documentation where this is explained could also be a good addition.

**Review by anfibiacreativa — CHANGES_REQUESTED** [review]
Thanks for you contribution! Please see the comment.

File: README.md (hunk 1 above)
Comment (ShatilKhan) [reply README.md:250]: No problem @anfibiacreativa,  thanks for noticing the screenshot.
Glad to contribute.
Should I add in the link to the docs still? 
Let me know.

Comment (anfibiacreativa) [reply README.md:250]: Yes, please! I think explaining exactly what error with auth the flag is connected to, is helpful for devs that may be facing other issues, not to get confused. Thank you!

Comment (ShatilKhan) [reply README.md:254]: Hey @anfibiacreativa , sorry for the delay.
As per your message, I added a link to the official doc & also a line explaining why we need the specific `flag`.
Please have a look & let me know.

Comment (anfibiacreativa) [reply README.md:254]: Oh I see now that you added them to this PR instead of the previous one? Is that correct?

Comment (ShatilKhan) [reply README.md:254]: Yes, sorry for the confusion. My Branches got mixed up accidently.
This commit is part of this PR only.
We don't need it in the previous one.

**Review by sinedied — APPROVED** [review]

**Review by anfibiacreativa — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**